src/NzbDrone.Core/Download/Clients/Nzbget/NzbgetSettings.cs

```diff
@@ -18,6 +19,8 @@ public NzbgetSettingsValidator()
 
             RuleFor(c => c.TvCategory).NotEmpty().When(c => !String.IsNullOrWhiteSpace(c.TvCategoryLocalPath));
             RuleFor(c => c.TvCategoryLocalPath).IsValidPath().When(c => !String.IsNullOrWhiteSpace(c.TvCategoryLocalPath));
+
+            RuleFor(c => c.TvCategory).NotEmpty().WithMessage("Using a separate category for Sonarr is recommended").AsWarning();
```

Comment:
Technically its required now, since CDH is broken without it. Ideally we require it when editing settings, but not when validating for using it (which was our issue with making it mandatory for existing users).